agent/utils/utils.go

```diff
@@ -199,3 +200,11 @@ func ParseBool(str string, default_ bool) bool {
 	}
 	return res
 }
+
+// IsAwsErrAndEqualToEcsErrCode returns true if the err implements
+// the Error interface of awserr and it has the same error code as
+// the passed in ecs error code.
+func IsAwsErrAndEqualToEcsErrCode(err error, ecsErrCode string) bool {
```

Comment:
As per https://github.com/golang/go/wiki/CodeReviewComments#initialisms `Aws` -> `AWS`, `Ecs` -> `ECS`. 

Also, I think `IsErrorCodeEqual(err error, code string)` seems much shorter and meaningful.

Reply:
ok, will change to `IsErrorCodeEqual(err error, code string)`